src/main/java/thrift/logic/parser/ListCommandParser.java

```diff
@@ -27,7 +28,7 @@ public ListCommand parse(String args) throws ParseException {
         if (!arePrefixesPresent(argMultimap, CliSyntax.PREFIX_MONTH)
```

Comment:
Instead of defining a new `arePrefixesPresent()` method, maybe you could use the existing method `ArgumentMultimap#getValue()` that returns Optional for checking presence of `Prefix`? E.g. `argMultimap.getValue(CliSyntax.PREFIX_VALUE).isPresent()`. But it's alright if you want to stick with `arePrefixesPresent()`.